Library/test-helpers/lib.sh

```diff
@@ -1884,7 +2045,93 @@ limeconRunAgent() {
     local AGENT_FILE=$5
     local TESTDIR=$6
 
-    podman run -d --name $NAME --net $NETWORK --ip ${IP} --privileged --volume=${AGENT_FILE}:/etc/keylime/ --volume=/sys/kernel/security/:/sys/kernel/security/:ro --volume=${TESTDIR}:${TESTDIR}:rw --ip ${IP} --device=/dev/tpmrm0  localhost/$TAG /usr/bin/keylime_agent
+    #mandatory for access containers to tpm
+    chmod o+rw /dev/tpmrm0
```

Comment:
Pls move this to the test itself so both `o+rw` and restore `o-rw` are at the same place, otherwise it would be confusing for a reader.

Reply:
Yeah, that could be more readable for others.